lib/double_entry/reporting/aggregate.rb

```diff
@@ -93,16 +95,17 @@ def range_is_complete?
 
       def field_hash
         {
-          :function => function,
-          :account => account,
-          :code => code,
-          :year => range.year,
-          :month => range.month,
-          :week => range.week,
-          :day => range.day,
-          :hour => range.hour,
-          :filter => filter.inspect,
-          :range_type => range.range_type.to_s,
+          :function        => function,
+          :account         => account,
+          :code            => code,
+          :partner_account => partner_account,
```

Comment:
Similar to [below](https://github.com/envato/double_entry/pull/108/files#r48796930), if `partner_account` is `nil` we'll only grab the `LineAggregate`s without a `partner_account`